syntax = "proto3";

message ProductRequest {
    int32 user_id = 1;
    int32 product_id = 2;
}

message ProductResponse {
    int32 status_code = 1;
}

service ProductService {
    rpc AddProduct (ProductRequest) returns (ProductResponse);
    rpc DeleteProduct (ProductRequest) returns (ProductResponse);
}